ui/packages/consul-ui/app/components/code-editor/README.mdx

```diff
@@ -6,6 +6,49 @@ state: needs-love
 # CodeEditor
 
 ```hbs preview-template
-<CodeEditor />
-
+<CodeEditor
+    @readonly={{true}}
+    @name={{concat name "[Rules]"}}
+    @syntax="hcl"
+    @oninput={{action "change" (concat name "[Rules]")}}
```

Comment:
Could we change this to:

```suggestion
    @oninput={{noop}}
```

That way when you go to the [Eng Docs] for this the page displays